assets/src/block-validation/sidebar.js

```diff
@@ -74,113 +49,36 @@ export function Sidebar() {
-					<div className="amp-sidebar__broken">
-						<div className="amp-sidebar__validation-errors-kept-icon">
-							<AMPValidationErrorsKeptIcon />
-						</div>
-						<div>
-							<h3>
-								{ __( 'Invalid markup kept', 'amp' ) }
-							</h3>
-							{ __( 'The permalink will not be served as valid AMP.', 'amp' ) }
-						</div>
-					</div>
-				</div>
-			) }
-			{ 0 < validationErrors.length && (
-				<PanelBody opened={ true } className="amp-sidebar__description-panel">
-					<div className="amp-sidebar__validation-errors-icon">
-						<AMPValidationErrorsIcon />
-					</div>
-					<div className="amp-sidebar__validation-errors-heading">
-						<h2>
-							{ __( 'Validation Issues', 'amp' ) }
-						</h2>
-
-						<p>
-							{ reviewLink && (
-								<ExternalLink href={ reviewLink } className="amp-sidebar__review-link">
-									{ __( 'View technical details', 'amp' ) }
-								</ExternalLink>
-							) }
-						</p>
-					</div>
-					{ hasReviewedValidationErrors && (
-						<div className="amp-sidebar__options">
-							<ToggleControl
-								checked={ isShowingReviewed }
-								label={ __( 'Include reviewed issues', 'amp' ) }
-								onChange={ ( newIsShowingReviewed ) => {
-									setIsShowingReviewed( newIsShowingReviewed );
-								} }
-							/>
-						</div>
-					) }
-				</PanelBody>
-			) }
+			<SidebarNotificationsContainer isShady={ true }>
+				<AMPRevalidateNotification />
+				<AMPValidationStatusNotification />
+			</SidebarNotificationsContainer>
 
-			{ isFetchingErrors ? (
-				<PanelBody opened={ true }>
-					<Loading />
-				</PanelBody>
-			) : (
-				<>
-					{ ! isPostDirty && validationErrors.length === 0 && (
-						<PanelBody opened={ true }>
-							<PanelRow>
-								<p>
-									{ isEditedPostNew
-										? __( 'Validation issues will be checked for when the post is saved.', 'amp' )
-										: __( 'There are no AMP validation issues.', 'amp' ) }
-								</p>
-							</PanelRow>
-						</PanelBody>
-					) }
-
-					{ isPostDirty && (
-						<PanelBody opened={ true }>
-							<PanelRow>
-								<p>
-									{ isEditedPostNew
-										? __( 'Validation issues will be checked for when the post is saved.', 'amp' )
-										: __( 'The post content has been modified since the last AMP validation.', 'amp' ) }
-								</p>
-							</PanelRow>
-							<PanelRow>
-								{ isDraft ? (
-									<Button isSecondary onClick={ () => savePost( { isPreview: true } ) }>
-										{ __( 'Save draft and validate now', 'amp' ) }
-									</Button>
-								) : (
-									<Button isSecondary onClick={ () => autosave( { isPreview: true } ) }>
-										{ __( 'Re-validate now', 'amp' ) }
-									</Button>
-								) }
-							</PanelRow>
-						</PanelBody>
-					) }
-				</>
+			{ 0 < displayedErrors.length && (
+				<ul className="amp-sidebar__errors-list">
+					{ displayedErrors.map( ( validationError ) => (
+						<li
+							key={ validationError.clientId }
```

Comment:
When I have a validation error coming from outside the content, I'm getting a React error:

![image](https://user-images.githubusercontent.com/134745/111583513-cf153c00-8779-11eb-8180-934e8232ba27.png)

Something like this seems necessary for errors that aren't connected with blocks:

```suggestion
					{ displayedErrors.map( ( validationError, i ) => (
						<li
							key={ validationError.clientId || i }
```

Reply:
I've addressed this issue in 9d35150. Initially, I didn't take into consideration the fact that some errors may have no `clientId` property (since they don't come from blocks).